sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -83,14 +84,18 @@ const (
 	URNArtifactURLType      = "beam:artifact:type:url:v1"
 	URNArtifactGoWorkerRole = "beam:artifact:role:go_worker_binary:v1"
 
-	// Environment Urns.
+	// Environment URNs.
 	URNEnvProcess  = "beam:env:process:v1"
 	URNEnvExternal = "beam:env:external:v1"
 	URNEnvDocker   = "beam:env:docker:v1"
 
-	// Userstate Urns.
+	// Userstate URNs.
 	URNBagUserState      = "beam:user_state:bag:v1"
 	URNMultiMapUserState = "beam:user_state:multimap:v1"
+
+	// Base version URNs are to allow runners to make distinctions between different releases
+	// in a way that won't change based on actual releases, in particular for FnAPI behaviors.
+	URNBaseVersionGo = "beam:version:sdk_base:go:" + core.DefaultDockerImage
```

Comment:
Is there a reason we're using the DefaultDockerImage as the identifier instead of the SdkVersion? They should be equally unique, but `SdkVersion` seems like a more natural identifier.

Reply:
I agree, but Java and Python went with the image URI for the URN instead. Better to be consistent.